solutions/ac-GCDOnBlackboard.mdx

```diff
@@ -112,6 +110,58 @@ ans = max(ans, prefix_gcd[n - 2])
 
 print(ans)
 ```
-
 </PySection>
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class GCDOnBlackboard {
+	public static void main(String[] args) throws IOException {
+		Kattio io = new Kattio();
+		int n = io.nextInt();
+		int a[] = new int[n];
+
+		for (int i = 0; i < n; i++) {
+			a[i] = io.nextInt();
+		}
+
+		// prefix and suffix gcd
+		int p[] = new int[n + 1];
+		int s[] = new int[n + 1];
```

Comment:
1. same thing with the array decs
2. instead of a comment you _could_ just name the variables `prefGCD` and `suffGCD`